corehq/apps/data_interfaces/signals.py

```diff
@@ -0,0 +1,23 @@
+from casexml.apps.case.models import CommCareCase
+from casexml.apps.case.signals import case_post_save
+
+from corehq.apps.data_interfaces.models import AUTO_UPDATE_XMLNS
+from corehq.form_processor.interfaces.dbaccessors import FormAccessors
+from corehq.toggles import AUTO_CASE_UPDATE_ENHANCEMENTS
+from corehq.form_processor.models import CommCareCaseSQL
+from corehq.form_processor.signals import sql_case_post_save
+
+
+def case_changed_receiver(sender, case, **kwargs):
```

Comment:
we should also wrap all of the code in this function into a try/except so that we never raise exceptions during the signal processing (django runs all signals synchronously). we can notify exception in the except clause